src/engine/shared/console.h

```diff
@@ -164,7 +138,7 @@ class CConsole : public IConsole
 	public:
 		CCommand *m_pCommand;
 		CResult m_Result;
-		CExecutionQueueEntry(CCommand *pCommand, CResult Result) :
+		CExecutionQueueEntry(CCommand *pCommand, CResult &Result) :
```

Comment:
```suggestion
		CExecutionQueueEntry(CCommand *pCommand, const CResult &Result) :
```

Commits can be squashed.

Reply:
This one or all of them?